solutions/secure-hybrid-network/nestedtemplates/azure-network-azuredeploy.json

```diff
@@ -522,12 +522,15 @@
                     }
                 ],
                 "sku": {
-                    "name": "Basic",
-                    "tier": "Basic"
+                    "name": "VpnGw2",
+                    "tier": "VpnGw2"
                 },
                 "gatewayType": "Vpn",
                 "vpnType": "RouteBased",
-                "enableBgp": false
+                "enableBgp": true,
+                "bgpSettings":{
+                    "asn": 60001
+                }
```

Comment:
Why is BGP now enabled?

Reply:
The error was related to that it was needed from the beginning 
https://github.com/mspnp/samples/issues/232
"The template output 'vpnIp' is not valid: The language expression property 'bgpSettings' doesn't exist"
bgpSettings doesn't exist because it was not enabled bgp. 
 Is this addressed in the Learn content? I don't know, I fixed the template execution.

Reply:
I mean, why `true`?

Reply:
Good catch. I thought it was mandatory to have bgpSettings, but I have just testing and it works with false.
Thanks.